e2e/__tests__/__snapshots__/watch_mode.test.js.snap

```diff
@@ -0,0 +1,64 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`can press "p" to filter by file name 1`] = `
+"
+Pattern Mode Usage
+ › Press Esc to exit pattern mode.
+ › Press Enter to filter by a filenames regex pattern.
+
+ pattern › [s[u pattern › b[s[u pattern › ba[s[u pattern › bar[s[u
```

Comment:
custom serializer or regexing to replace `[s[u` with a newline?

Reply:
👏 👏 👏